sdk/cosmos/azure-cosmos/azure/cosmos/_service_request_retry_policy.py

```diff
@@ -11,22 +11,25 @@
 from azure.cosmos.documents import _OperationType
 from azure.cosmos.http_constants import ResourceType
 
+# This Retry policy is for the requests which were never sent to the service.
```

Comment:
Just curious - should these changes reflect instead the diff with `4.9.1b2` and not main?